### DISCUSSION TITLE
Live status and user cards not loading as anonymous

### DISCUSSION THREAD

**Discussion by Romybron** [Chatterino/chatterino2#5743] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

But it works when login in then going anonymous, I am guessing it simply reuses the user token.

An idea to solve this would be to use a saved and working user token when opening Chatterino without connecting to chat with it, what do you think?

### OS and Chatterino Version

Chatterino 2.3.5 (commit 81a62764c)

**Comment by Mm2PL** [comment]
Works as intended. Or rather as Twitch intended, we can't do any better. This used to work when Kraken API was around, but Twitch has since removed the ability to use the API with only Client-ID. An app token is very impractical and an API proxy is probably* against the Twitch Developer Agreement because of the forwarding Twitch information clause and isn't any more practical than an app token.

*IANAL

Edit: I remember suggestions to use an existing token from the app but that IMO violates the whole idea of a mode where you aren't logged in at all. I can't find them though.

**Comment by Romybron** [comment]
I don't really get what you mean. I may not have made myself clear? It would only required to use any of the saved user tokens for the anonymous. This is basically the same behavior than login in with username then going anonymous. But now it would do that when launch the app also. It wouldn't require any proxy or anything, just verify and use the token without connecting to IRC with it. This wouldn't break TOS since the token can be use with or without IRC.

**Comment by nonua** [comment]
@Romybron I get what you mean. Sometimes I close Chatterino in anonymous mode and when opening it the next day, tabs don't show display names, live pings don't get through, there is no live statutes and info about stream, user cards have no info, and probably other Helix and PubSub related stuff.

When I want to stay anonymous and benefit all those things, I first need to login, close Chatterino, open Chatterino, and then logout. This will make everything work normally and stay anonymous.

@Mm2PL Any second thoughts ?

**Comment by Mm2PL** [comment]
I guess we should reconsider a non-truly anonymous mode

**Comment by pajlada** [comment]
I've converted this to a discussion to figure out a bit more in-depth why this is useful, other than "i like switching to anonymous mode", before approving this as a feature
